flocker/node/agents/ebs.py

```diff
@@ -174,7 +174,6 @@ def _blockdevicevolume_from_ebs_volume(ebs_volume):
 
     :return: Input volume in BlockDeviceVolume format.
     """
-    ebs_volume.update()
```

Comment:
This makes sense to me but I suspect it will break some other code.

`create_volume` does `requested_volume = self.connection.create_volume(...)` and later `return _blockdevicevolume_from_ebs_volume(requested_volume)`.  `_blockdevicevolume_from_ebs_volume` uses `ebs_volume.tags[DATASET_ID_LABEL]`.  `create_volume` only sets that tag after the volume is created.  So I'd expect `requested_volume` to be missing the information.  Perhaps boto is extra-mutate-y and makes the tag available anyway somehow, I'm not sure.    BuildBot hasn't caught up yet so I'm not sure what the test suite has to say about this.